node/service/Cargo.toml

```diff
@@ -140,17 +140,27 @@ substrate-test-runtime-client = { git = "https://github.com/paritytech/substrate
 
 
 [features]
-default = [ "kusama-native", "westend-native", "rococo-native" ]
+default = [
+	"kusama-native",
+	"westend-native",
+	"rococo-native",
+	"moonbase-runtime",
+	"moonriver-runtime",
+	"moonbeam-runtime"
+]
+
 kusama-native = [ "polkadot-service/kusama-native", "polkadot-cli/kusama-native" ]
 westend-native = [ "polkadot-service/westend-native", "polkadot-cli/westend-native" ]
 rococo-native = [ "polkadot-service/rococo-native", "polkadot-cli/rococo-native" ]
```

Comment:
I think we can do better than this. It isn't just the Moonbeam runtimes that are heavy to build, but also the polkadot runtimes. We should make three features, called something like `moonbeam-native`, `moonriver-native`, and `moonbase-native` (or something similar). And when those features are enabled, they pull in the corresponding polkadot runtimes. So for example, when we enable the `moonriver-native` feature, it also pulls in the `kusama-native` feature from polkadot crates.

Reply:
After analysis, it seem's that we never need `rococo-native` and `kusama-native`. And we need `westend-native` only with moonbase runtime.